assets/js/modules/search-console/util/index.js

```diff
@@ -58,25 +58,15 @@ function reduceSearchConsoleData( rows ) {
 		totalPosition += row.position;
 	} );
 
-	const totalClicksRaw = totalClicks;
-	const totalImpressionsRaw = totalImpressions;
-
-	totalClicks = readableLargeNumber( totalClicks );
-	totalImpressions = readableLargeNumber( totalImpressions );
-
 	// Do not divide by zero.
-	const averageCTR = count > 0 ? ( totalCTR / count * 100 ).toFixed( 1 ) : '0.0';
-	const averageCTRRaw = count > 0 ? totalCTR / count : 0.0;
-	const averagePosition = count > 0 ? ( totalPosition / count ).toFixed( 1 ) : '0.0';
+	const averageCTR = count > 0 ? totalCTR / count : 0.0;
+	const averagePosition = count > 0 ? +( totalPosition / count ).toFixed( 1 ) : 0.0;
```

Comment:
We were doing this before but I don't think we should be truncating or otherwise pre-formatting the value here as this should be handled by the formatting function. Unless we specifically care about doing the math this way, we should leave formatting to the utilities.

Reply:
@aaemnnosttv, I've kept this as this logic demands only one fraction digit. i.e 1.99 should be displayed as 1.9% and not 1.99%.
In the new `numFmt` implementation the default number of fraction digits is 2 therefore we need this logic. If you can approve that having 2 fraction digits is fine in terms of the UI then I can go ahead and remove this logic.

Reply:
I would think that the default number of fractional digits would be okay here but I think that would be a change @felixarntz would have to approve.

My point still stands however that we should not be manipulating the numbers here to achieve a specific formatted result. If this number should be formatted in a specific way which is different than the default, then it should be possible to pass this via props to pass different `options` to `numFmt`.

Reply:
I think we should update the `metric` format to allow for a maximum of 1 fraction digit. This maintains the formatting as present, and it also matches the rest of the behavior of that function (where there's never more than 1 digit).

Reply:
@felixarntz the style in question here is `percent` and not `metric`. There probably isn't any issue with number of fraction digits for `metric` or `decimal` style. 
The problem here is related to the UI issues we might face in the concerned components if we use default behavior of `DataBlock`'s `datapoint` formatting logic i.e. percentage can have maximum of 2 fraction digits.
We have these options here:
* We can either send a number with only one fraction digit here 
* allow overloading `datapointUnit`/`changeUnit` props to accept `object` which will be passed to `numFmt`.
* be fine with 2 fraction digits.

Reply:
@kostyalmm The value in question here is the `averagePosition` (not the `averageCTR` which is also shown in the diff), and that one doesn't pass `datapointUnit`, i.e. uses `metric`. So I think that is correct. The `averageCTR` uses a regular percentage formatting and doesn't have the `toFixed` part (as expected).